src/core/qtlibtorrent/qbtsession.cpp

```diff
@@ -1858,7 +1858,7 @@ void QBtSession::setListeningPort(int port) {
   if (iface_name.isEmpty()) {
     logger->addMessage(tr("qBittorrent is trying to listen on any interface port: %1", "e.g: qBittorrent is trying to listen on any interface port: TCP/6881").arg(QString::number(port)), Log::INFO);
     if (listen_ipv6)
-      s->listen_on(ports, ec, "::", session::listen_no_system_port);
+      s->listen_on(ports, ec, 0, session::listen_no_system_port);
```

Comment:
In the special case of "any interface" this should be used regardless of listen_ipv6.